.travis.yml

```diff
@@ -34,6 +34,9 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools --depth=1
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - pip install python-ldap
+  - pip install unidecode
+  - pip install validate_email
```

Comment:
OK. With the new support for requirements.txt on repos this will be unnecessary once the requirements file is added to product-attribute.